frontend/src/views/FormView.vue

```diff
@@ -211,13 +222,17 @@ const styleFailed = computed(() => styleLogs.value.some((l) => l.status != 'OK')
         >
           <div class="my-1">{{ $t('Source:') }} {{ geonetwork.src }}</div>
           <div class="my-1">{{ $t('Destination:') }} {{ geonetwork.dst }}</div>
-          <div class="my-1">
-            {{ $t('Metadata:') }}{{ metaSuccessful ? ' ✅' : metaFailed ? ' ❌' : '' }}
-          </div>
-          <ul v-for="(metadata, index) in geonetwork.metadata" :key="index">
-            <li class="list-disc ml-4">{{ metadata.title }}</li>
-          </ul>
-          <LogsReport v-if="logs.length" :logs="metaLogs"></LogsReport>
+          <Panel toggleable :collapsed="true">
+            <template #header>
+              <div>
```

Comment:
This div seems useless.

Note that I plan to remove other tags, but didn't do it yet to avoid conflicts.

Reply:
@arnaud-morvan : no, this div is important, be carful if you remove it. The contents of the Panel header is a horizontal flex-box. So I had to add the div to break the flex-box layout.

Reply:
(otherwise the UL list ended up on the right of the title text)